tests/test_output.py

```diff
@@ -95,6 +96,7 @@ def tearDown():
                 os.unlink(join(config_dir, cf))
 
         for output_file in (glob(join(output_dir, 'test_{}_*'.format(suffix))) +
+                            glob(join(output_dir, 'test_{}.log'.format(suffix))) +
```

Comment:
Is this actually removing all of the outputs or just the `.log` file? You probably need the asterisk here like the others.

Reply:
Actually, `test_{}_*` was not removing log files which would have format `test_{}.log`. So, I added this extra logic to remove log files.

Reply:
But are there other files named `test_{}` (without the second underscore) that also need to be removed but aren't? Should you change `test_{}.log` to `test_{}.*`?

Reply:
I did not see any other files other than log. But, if there are, then they should be removed too. Let me check if any.